api-js/src/affiliation/mutations/__tests__/update-user-role.test.js

```diff
@@ -892,7 +894,7 @@ describe('update a users role', () => {
 
           const error = [
             new GraphQLError(
-              'Unable to update users role. Please invite user to the organization.',
+              'Unable to update users role that does not belong to this org. Please invite user to the organization.',
```

Comment:
-> "Unable to update role of user that does not belong to this org" ?

Reply:
"Unable to update role of user that does not belong to this org"

This sounds.....weird?  "Unable to update role: user does not belong to organization"  Or something